docs/pages/admin-guides/management/guides/scim-sailpoint.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: SCIM Sailpoint Integration
+description: How to manage Access List membership via SCIM SailPoint integration
+
+---
+
+The SCIM integration between **SailPoint** and **Teleport** enables automated
+synchronizing SailPoint group memberships with
+Teleport Access Lists. This integration supports centralized identity governance in
+SailPoint while Teleport enforces fine-grained access controls.
+
+User permissions in Teleport are defined through Access Lists. While role definitions
+live in Teleport, **group membership is dynamically managed** by SailPoint SCIM group
```

Comment:
```suggestion
live in Teleport, group membership is dynamically managed by SailPoint SCIM group
```
I don't think we need bold text here, since the bolded text makes up a substantial part of the sentence, and we tend to only bold introductions of new keywords.